boards/common/blxxxpill/Makefile.include

```diff
@@ -8,20 +8,15 @@ ifeq (dfu-util,$(PROGRAMMER))
 
   # Leave some space at the beginning of the flash for the bootloader
   ROM_OFFSET ?= 0x2000
-
+  DFU_USE_DFUSE = 1
```

Comment:
This line could be moved after `DFU_ALT`. This way DFU related variables are grouped.

Reply:
Done